app/views/publish_data/pages/service_manual.html.erb

```diff
@@ -47,7 +47,9 @@
       <div class="markdown">
         <h1 class="heading-large" id="what-is-service-performance">What is Service Performance?</h1>
         <p>
-          Service Performance is a new tool from GDS that displays KPIs from government services. It's designed to give civil servants working in service transformation information about how services are performing.
+          Service Performance is a new tool from GDS that displays metrics from government services.
+          
+          It's for anyone who needs information on how services are performing.
         </p>
         <h2 class="heading-large" id="kpis">
```

Comment:
Could you please change `id="kpis"` to `id="metrics"`? That will make sure that the metrics link in the contents links to here.